manifests/install.pp

```diff
@@ -1,6 +1,13 @@
 # Install the needed packages for foreman
 class foreman::install {
 
+  if $foreman::db_manage and $facts['os']['family'] == 'RedHat' {
```

Comment:
I think it's pretty simple to re-work this for other OSes as well if needed, but what about other locales? We seem to be declaring `postgresql::server::db` with `en_US.utf8` hardcoded so I think I can ignore them?

Reply:
I agree. The DB is defined as en_US so this needs to be present.